interface/therapy_groups/therapy_groups_views/listGroups.php

```diff
@@ -0,0 +1,399 @@
+<?php
+/**
+ * interface/therapy_groups/therapy_groups_views/listGroups.php contains the group list view .
+ *
+ * In this view all therapy groups are listed with their details and links to their details screen.
+ *
+ * Copyright (C) 2016 Shachar&Amiel <shachar058@gmail.com> <amielboim@gmail.com>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Shachar Zilbershlag <shachar058@gmail.com>
+ * @author  Amiel Elboim <amielboim@gmail.com>
+ * @link    http://www.open-emr.org
+ */
+?>
+<?php require 'header.php'; ?>
+
+<div id="therapy_groups_list_container" class="container">
+
+    <!--------- ERRORS ----------->
+    <?php if($deletion_try == 1 && $deletion_response['success'] == 0) :?>
+        <div class="row">
+            <div class="col-md-6 col-md-offset-3">
+                <div class="alert alert-danger text-center">
+                    <p class="failed_message"><?php echo xlt($deletion_response['message']);?></p>
```

Comment:
What is an example of the $deletion_response['message']?
(seeing if it needs to be translated)